xml/System.IO/WindowsRuntimeStorageExtensions.xml

```diff
@@ -52,6 +52,28 @@ The next example shows the XAML code that is associated with the previous exampl
     </remarks>
   </Docs>
   <Members>
+    <MemberGroup MemberName="CreateSafeFileHandle">
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime.WindowsRuntime</AssemblyName>
+        <AssemblyVersion>4.0.14.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Docs>
+        <summary>Creates and returns a safe file handle for the IStorage instance that is being extended.</summary>
```

Comment:
This should be a reference to SafeFileHandle

Reply:
```suggestion
        <summary>Creates and returns a reference to a safe file handle for the IStorage instance that is being extended.</summary>
```